v2.0.0/tasks-schema.json

```
@@ -0,0 +1,1338 @@
+                                                    "2022-11-19"
+                                                ]
+                                            }],
+                                            "type": "object",
+                                            "properties": {
+                                                "required": {
+                                                    "description": "Array of origin date unique identifiers that must be present for submission to be valid. Can be null if no origin dates are required and all valid origin dates are specified in the optional property.",
+                                                    "type": [
+                                                        "array",
+                                                        "null"
+                                                    ],
+                                                    "uniqueItems": true,
+                                                    "items": {
+                                                        "type": "string",
+                                                        "format": "date"
+                                                    }
+                                                },
+                                                "optional": {
+                                                    "description": "Array of valid but not required unique origin date identifiers. Can be null if all origin dates are required and are specified in the required property.",
+                                                    "type": [
+                                                        "array",
+                                                        "null"
+                                                    ],
+                                                    "uniqueItems": true,
+                                                    "items": {
+                                                        "type": "string",
+                                                        "format": "date"
+                                                    }
+                                                }
+                                            },
+                                            "required": [
+                                                "required",
+                                                "optional"
+                                            ]
+                                        },
+                                        "forecast_date": {
+                                            "description": "An object containing arrays of required and optional unique forecast dates. Forecast date usually defines the starting point that can be used for calculating a target_date via the formula target_date = forecast_date + horizon x time_units_per_horizon (e.g., with weekly data, target_date is calculated as forecast_date + horizon x 7 days)",
```

Historically, a forecast date has been the date that a forecast was run or submitted to a hub, which is different than a kind of reference date or origin date for calculating target dates. I'm editing this description here to reflect the way this has been used in the past.
```suggestion
                                            "description": "An object containing arrays of required and optional unique forecast dates. Forecast date usually defines the date that a model is run to produce a forecast.",
```

---

@elray1 's suggestion looks good to me.